packages/grafana-ui/src/components/OptionsUI/string.tsx

```diff
@@ -37,3 +43,21 @@ export const StringValueEditor: React.FC<FieldConfigEditorProps<string, StringFi
     />
   );
 };
+
+export const StringWithSuggestionsValueEditor: React.FC<FieldConfigEditorProps<
```

Comment:
is it possible to avoid a new editor?  just use the existing text one?

In general, it should be the exception that we do *not* support template variables